.circleci/config.yml

```diff
@@ -756,7 +756,11 @@ jobs:
       - run:
           name: Delete Source API Gateway Mappings
           command: |
-            npm run delete:api-gateway-mappings -- $EFCMS_DOMAIN $DEPLOYING_COLOR
+            if [ "${CIRCLE_BRANCH}" == "prod" ]; then
```

Comment:
Nit: The job `backup-source-dynamo-table` is only run in the prod deploy workflow so it may be worth it to rename that to be a bit less specific and move this into that job as well? That way we could make use of the built in branch filtering Circle provides

Reply:
Are you sure?  🤔  I only see `backup-source-dynamo-table` having its steps defined, stating that it requires `switch-colors` to be run first, and that it is required before `backup` runs.  I don't see it having a CircleCI branch filter.

Reply:
Ah, I see!

The filter isn't specified on the workflow, but on the job level still.  It just so happens that _every_ job in the `build-and-deploy-with-context` workflow has the `build-and-deploy-with-context-defaults` settings injected into it.  It is in `build-and-deploy-with-context-defaults` that has the `prod` branch filter.

Thanks for pointing this all out.  I'll look into it!

Reply:
I ended up making the deletion of the API Gateway mappings its own job that only runs in production.  I updated the description of this PR with additional detail.